src/views/Dashboard/Customer/Home/index.vue

```diff
@@ -208,9 +208,11 @@ export default {
       try {
         this.testHistory = [];
         let maxResults = 5;
+        const dummyAddress = "5GH6Kqaz3ZewWvDCZPkTnsRezUf2Q7zZ5GmC4XFLNqKdVwA7"
```

Comment:
are you still using dummy address? can't we proceed wiring to real data instead?

Reply:
done i already change to real address